packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js

```diff
@@ -10,13 +10,13 @@ function ItemsNumber(props) {
 			{selected
 				? labelSelected ||
 				  t('LIST_TOOLBAR_NUMBER_OF_SELECTED_ITEMS', {
-						defaultValue: '{{count}}/{{total}} items',
+						defaultValue: '{{count}}/{{total}} item',
 						count: selected,
 						total: totalItems,
 				  })
 				: label ||
 				  t('LIST_TOOLBAR_TOTAL_NUMBER_OF_ITEMS', {
-						defaultValue: '{{count}} items',
+						defaultValue: '{{count}} item',
```

Comment:
No more `s` 🤔

Reply:
correct! The plural is added in LIST_TOOLBAR_TOTAL_NUMBER_OF_ITEMS_plural.

Reply:
added defaultValue_plural